docs/commands.md

```diff
@@ -234,3 +234,42 @@ If needed, input tensors are copied to the device specified in `AI.SCRIPTSET` be
 ```sql
 AI.SCRIPTRUN addscript addtwo INPUTS a b OUTPUTS c
 ```
+
+## AI.INFO
+
+Return information about runs of a `MODEL` or a `SCRIPT`.
+
+At each `MODELRUN` or `SCRIPTRUN`, RedisAI will collect statistcs specific for each `MODEL` or `SCRIPT`,
+specific for the node (hence nodes in a cluster will have to be queried individually for their info).
+The following information is collected:
+
+- `KEY`: the key being run
+- `TYPE`: either `MODEL` or `SCRIPT`
+- `BACKEND`: the type of backend (always `TORCH` for `SCRIPT`)
+- `DEVICE`: the device where the run has been executed
+- `DURATION`: cumulative duration in microseconds
+- `SAMPLES`: cumulative number of samples obtained from the 0-th (batch) dimension (for `MODEL` only)
+- `CALLS`: number of calls
+- `ERRORS`: number of errors generated after the run has been submitted (i.e. excluding errors generated during parsing of the command)
+
+```sql
+AI.INFO <model_or_script_key>
+```
+
+Statistcs are accumulated until the same command with an extra `RESETSTAT` argument is called. This resets the statistics relative to the model or script.
+
+```sql
+AI.INFO <model_or_script_key> RESETSTAT
+```
+
+The command can be called on a key until that key is removed using `MODELDEL` or `SCRIPTDEL`.
+
+### AI.INFO Example
+
+```sql
+AI.INFO amodel
```

Comment:
@lantiga can we include the example output here?

Reply:
Done